### ISSUE TITLE
[steamshovel] Detector.cpp does not know about scintillators (Trac #1749)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#1741]
Line 129 in private/shovelart/artists/Detector.cpp:

```text
		if( icetop ){
			if (hide_icetop )
				continue;
		}else{
			if( hide_icecube )
				continue;
		}
```

Migrated from https://code.icecube.wisc.edu/ticket/1749
```json
{
    "status": "closed", 
    "changetime": "2019-02-13T14:12:47", 
    "description": "Line 129 in private/shovelart/artists/Detector.cpp:\n{{{\n\t\tif( icetop ){\n\t\t\tif (hide_icetop )\n\t\t\t\tcontinue;\n\t\t}else{\n\t\t\tif( hide_icecube )\n\t\t\t\tcontinue;\n\t\t}\n}}}\n", 
    "reporter": "pmeade", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1550067167842669", 
    "component": "combo core", 
    "summary": "[steamshovel] Detector.cpp does not know about scintillators", 
    "priority": "major", 
    "keywords": "", 
    "time": "2016-06-16T21:51:44", 
    "milestone": "", 
    "owner": "hdembinski", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20160617T12:38:43: 
* hdembinski commented:

> In changeset 147543:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="147543"
> include scintillators into the group of icetop doms, closes #1741
> ```


* hdembinski changed resolution from "" to "fixed"
* hdembinski changed status from "new" to "closed"

**Comment by icecube-trac** [comment]
Trac update at 20190213T14:12:47: 
* olivas changed milestone from "June 2016" to ""
* olivas commented:

> Milestone June 2016 deleted
